distribution/kernel/carbon-home/dbscripts/mysql.sql

```diff
@@ -260,9 +260,6 @@ CREATE TABLE UM_DOMAIN(
             UNIQUE(UM_DOMAIN_NAME,UM_TENANT_ID)
 )ENGINE INNODB;
 
-CREATE UNIQUE INDEX INDEX_UM_TENANT_UM_DOMAIN_NAME
```

Comment:
is this expected here? we are removing some index but couldn't find the equivalent logics

Reply:
The unique index is defined for this column in `UM_TENANT` table at https://github.com/wso2/carbon-kernel/pull/4185/files/4cd903198485e4e892b49ad37cfbb936851a30f7#diff-8e4b6f6e9687671294c7a0cc82c9c1dac1cb7608da97b928bb01136d4d3cf065R249.